CIP-0030/README.md

```diff
@@ -200,6 +200,13 @@ Upon start, dApp can explicitly request a list of additional functionalities the
 
 DApps are expected to use this endpoint to perform an initial handshake and ensure that the wallet supports all their required functionalities. Note that it's possible for two extensions to be mutually incompatible (because they provide two conflicting features). While we may try to avoid this as much as possible while designing CIPs, it is also the responsability of wallet providers to assess whether they can support a given combination of extensions, or not. Hence wallets aren't expected to fail should they not recognize or not support a particular combination of extensions. Instead, they should decide what they enable and reflect their choice in the `cardano.{walletName}.extensions` field of the Full API. As a result, dApps may fail and inform their users or may use a different, less-efficient, strategy to cope with a lack of functionality.
 
+Extension's endpoints should be namespaced by `.cipXXXX.` with the returned `API` object. For example; CIP-1234's endpoints should be accessed by:
+
+```ts
+api.cip1234.endpoint1()
+api.cip1234.endpoint2()
+```
+
```

Comment:
I would be clear here that the cipxxxx is not using leading zeros.  Otherwise is it:
`api.cip0095.endpoint()`
or 
`api.cip95.endpoint()`

I think trimming leading zeros is both easier to read and less open to interpretation.

Reply:
Good call, I have made this explicit in [bc52762](https://github.com/cardano-foundation/CIPs/pull/577/commits/bc52762bca6bc12a65d6a1c9905e0b1c9228445f).